### PR TITLE
Task 220 -- Story Hub Issues

### LINKED ISSUE #220: [Stories] Hub Page remaining issues
- [x] Filtering refactoring **( AMLAN )**
- [x] Filters need to be in alphabetical order unless specified by the business **( AMLAN )**
- [x] Push the card listing down so it sits parallel to the facets **( AMLAN )**
- [x] Update the publication type text color to 65697C instead of #000000 and font to rockwell **( AMLAN )**
- [x] Apply 2 column/cards for resolution between 576px and 1024px **( AMLAN )**
- [x] Text color for the card's CTA needs to be corrected. Please update it to Abcam's green - #378189 **( AMLAN )**
- [x] Reduce the horizontal gap between cards/columns to 24px **( AMLAN )**
- [x] Font size for card sub text is incorrect **( AMLAN )**
- [x] Make the filters heading also clickable rather than limiting it only to the chevron icon **( AMLAN )**
- [x] The chevron icon for filters section. In collapsed state, it should be facing downwards and vice-versa **( AMLAN )**
- [x] On mobile and tablet devices, a black solid color appears around the image rather than covering it **( AMLAN )**
- [x] Sort all/filtered articles based on `publishDate`, from newest-date to oldest-date **( AMLAN )**
- [x] Carousal Auto Play issue **( SHIVANGI )**
- [x] Trying to scroll the page with the mouse cursor on the carousel, the hero image and carousel text scroll instead of the page **( SHIVANGI )**
- [x] Increase the page heading/title font size to 3.5rem (desktop) / 2.5 rem(mobile and tablet) and  add 40px gap between title and carousel **( SHIVANGI )**

### PR DESCRIPTION
**Pull request by amlan18041996**
Fix #220 

Test URLs:
- Before: https://main--danaher-abcam--aemsites.hlx.page/en-us/stories
- After: https://220-hub-page-remaining-issues--danaher-abcam--aemsites.hlx.page/en-us/stories

### CODE REVIEW

**Comment by amlan18041996** [comment]
Adding as a **draft** as the filter changes needs to be discussed.

**Review by shivangi1422 — APPROVED** [review]

File: eds/blocks/related-stories/related-stories.js [hunk 1]
```diff
@@ -57,7 +57,7 @@ function createCard(article, firstCard = false) {
       div(
         {
           class:
-            'mt-auto inline-flex w-full pb-5 text-base text-[#378089] font-semibold',
+            'mt-auto inline-flex w-full pb-5 text-base text-[#378189] font-semibold',
```
Comment (rgravitvl) [inline eds/blocks/related-stories/related-stories.js:60]: Add the color code to tailwind config

**Review by rgravitvl — APPROVED** [review]
One minor comment, rest looks good

**Review by pardeepgera23 — APPROVED** [review]

**Review by sharanagoudapatil — APPROVED** [review]

File: eds/blocks/related-stories/related-stories.js (hunk 1 above)
Comment (pardeepgera23) [reply eds/blocks/related-stories/related-stories.js:60]: @amlan18041996  - Please create a new task for moving all the color codes to tailwind config and get the color names mapping from Sunny.
cc: @rgravitvl @sharanagoudapatil @shivangi1422

**Pull request merged**